### DISCUSSION TITLE
Can I use the wallet to mine the data blocks that are generated by the P disk software of the mining pool in the future?

### DISCUSSION THREAD

**Discussion by ndcc5** [Chia-Network/chia-blockchain#2097] [Q&A]
Can I use the wallet to mine the data blocks that are generated by the P disk software of the mining pool in the future? 
I hope the official will give a clear explanation, P disk is not easy.
Send the P disk command of the mining pool to everyone to take a look. With Poolkey inside. I don't know if it affects solo. .

Send the P disk command of the mining pool to everyone to take a look. With Poolkey inside. I don't know if it affects solo. .
**chia-plotter-windows-amd64 -action plotting -plotting-fpk 0x12345  -plotting-ppk 0x12345 -plotting-n 1 -d d:\ -t d:\
pause**

![image](https://user-images.githubusercontent.com/6286019/115191650-4db81d00-a11c-11eb-9e4d-cf0f269e6033.png)

**Comment by cccat6** [comment]
I don't really understand what you are trying to ask. 我没看懂你的问题
I will guess it anyway. 所以我猜一下你的意思
Your setup in the screenshot is half correct. 你的截图里配置有一部分正确，一部分有问题
All the keys should not contain "0x" at the beginning. It just references it as HEX. You should get the keys and fingerprint with `chia keys show` 所有key的开头不需要加“0x”，这只是表示它是十六进制而已。你应该在Chia的主程序目录执行`chia keys show`来获取需要的key和指纹
If anyone plotting with your parms. The revenue of farming these plots will goes to your wallet. 如果别人用你的参数来p盘，生成的plot文件是属于你的。因为挖这些plot产生的收益会到你的钱包里。
If you are asking actual pooling but not solo work, I am unable to answer since Chia has not officially release pooling. 如果你问的是怎么进矿池，而不是solo的话，我回答不了你。官方到现在还没有放出关于矿池的信息，你可以等等看。
